### ISSUE TITLE
Chore: Improve Accessibility

### ISSUE THREAD

**Issue by nelsonic** [dwyl/mvp#129]
At present the Accessibility of the MVP is not lacking:
https://web.dev/measure/?url=https%3A%2F%2Fmvp.fly.dev%2F
<img width="1098" alt="image" src="https://user-images.githubusercontent.com/194400/184469654-7a0bcea7-9fb3-45f9-ad6d-f970b8cec42c.png">

This is reasonably easy to solve as we only have one form. 

# Todo

+ [x] Follow the guidelines on Accessibility to achieve a 100% score.

## Best Practices

+ [x] Check best practices advice on logo image size. Can we create a smaller logo e.g. **`24x24px`**
  ***or*** enlarge the logo in the UI to **`32x32px`** 

<img width="992" alt="image" src="https://user-images.githubusercontent.com/194400/184470086-dcf3a704-af52-48a7-a2c8-5815df435c2e.png">

At present this is what the Top Nav looks like for me:
<img width="754" alt="image" src="https://user-images.githubusercontent.com/194400/184470115-0c72b17d-bff8-46f2-b7f6-43a8c92a66e7.png">

Related: https://github.com/dwyl/app-mvp/issues/128

**Comment by nelsonic** [comment]
I'm going to try and pick this up now as my last pomodoro of the day. ⏳ 🍅

**Comment by nelsonic** [comment]
![image](https://user-images.githubusercontent.com/194400/184552134-d4f40989-333f-46dd-ae6e-7b457a16c821.png)

**Comment by nelsonic** [comment]
First thing on the Accessiblity Improvement list is: "_Form elements do not have associated labels_"

![image](https://user-images.githubusercontent.com/194400/184552363-95954799-6b2a-4f3c-be4b-11666176143b.png)

Docs: https://web.dev/label/#:~:text=Labels%20ensure%20that%20form%20controls,makes%20a%20larger%20click%20target.

Question: https://www.google.com/search?q=can+a+form+label+be+hidden

https://www.w3.org/WAI/tutorials/forms/labels/#:~:text=The%20label%20can%20be%20hidden,reader%20and%20speech%20input%20users.
![image](https://user-images.githubusercontent.com/194400/184552390-7bde2021-54c9-4153-9ffc-4de869d0ba6e.png)

Ok, that was easy enough. 👌

**Comment by nelsonic** [comment]
Next is a bit more tricky because it kinda messes with the "branding" ... 💭 🌈 
![image](https://user-images.githubusercontent.com/194400/184552516-43b2dbdf-de1a-43d6-83c1-2b1f05296007.png)

Going to try and increase the contrast by bumping the saturation on all the buttons. 🧑‍💻

**Comment by nelsonic** [comment]
Definitely prefer the brighter colors ... but this is what I've done:

![image](https://user-images.githubusercontent.com/194400/184553598-10e7700d-b905-4174-bb2b-5237a09449a0.png)

**Comment by nelsonic** [comment]
Going through the errors detected by Validator:
https://validator.w3.org/nu/?doc=https%3A%2F%2Fmvp.fly.dev%2F
![image](https://user-images.githubusercontent.com/194400/184553951-f9c4b645-738a-4815-b478-fa73235f854f.png)

**Comment by nelsonic** [comment]
Basically _none_ of the custom `LiveView` attributes e.g. `phx-track-static`, `phx-submit`, `phx-value-id`
or `x-on:input` from Alpine.js are allowed by W3C ... 🤦‍♂️

**Comment by nelsonic** [comment]
Deployed my branch and now get: https://web.dev/measure/?url=https%3A%2F%2Fmvp.fly.dev%2F
![image](https://user-images.githubusercontent.com/194400/184554726-5d1efde8-8309-48ca-ab10-43f9efc15ed6.png)

PR: #130

**Comment by nelsonic** [comment]
Reading: https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP

**Comment by nelsonic** [comment]
https://content-security-policy.com/

**Comment by nelsonic** [comment]
Looks like we need to add `'unsafe-eval'` for the scripts loaded from `cdnjs` ... 💭 
see: https://github.com/aframevr/aframe/issues/5028

**Comment by nelsonic** [comment]
Accessibility 100. ✅ 
![image](https://user-images.githubusercontent.com/194400/184555641-f3167c18-1840-4799-8eb6-dd2a3081be69.png)

**Comment by nelsonic** [comment]
The final aspect of "Best Practices" that we can improve on is the images ... 
https://web.dev/codelab-serve-images-correct-dimensions/

Think it's worth having a _separate_ issue for this. 💭

**Comment by nelsonic** [comment]
https://elixirforum.com/t/securityerror-the-operation-is-insecure-content-security-policy-directive-default-src-self/48362/10

**Comment by nelsonic** [comment]
Don't think there's anything `else` to do for this issue: https://web.dev/measure/?url=https%3A%2F%2Fmvp.fly.dev%2F
<img width="1087" alt="image" src="https://user-images.githubusercontent.com/194400/185811152-cc3504a0-df66-49f5-8061-86357da5a388.png">

**Comment by nelsonic** [comment]
Closing. ✅
